synapseclient/__main__.py

```diff
@@ -631,8 +621,6 @@ def build_parser():
                                 ' types to not copy.')
     parser_cp.add_argument('--skipCopyWiki', action='store_true',
                            help='Do not copy the wiki pages')
-    parser_cp.set_defaults(func=copy)
```

Comment:
please also remove other `parser_cp.<add_argument/other funcs>` that set arguments or other attribute to the command line doc.